documentation/docs/14-debugging.md

```diff
@@ -1,19 +0,0 @@
----
-title: Debugging
----
-
-Debugging your server code is particularly easy with [ndb](https://github.com/GoogleChromeLabs/ndb). Install it globally...
```

Comment:
I've never used `ndb` before, but I guess our opinion has changed on it if we used to recommend it and no longer do?

Reply:
i don't know that i've ever used it. it hasn't been updated in 2 years, and a lot has changed in node since then, so i for one don't feel all that comfortable vouching for it